README.md

```diff
@@ -83,7 +83,10 @@ Example of configuration **RADIUS**:
                 {link_broken, 2},
                 {upf_failure, 9},
                 {remote_failure, 9},
-                {inactivity_timeout, 4},
+                {cp_inactivity_timeout, 4},
```

Comment:
Please update also `METRICS.md` and add:
```
 * cp_inactivity_timeout
 * up_inactivity_timeout
 * aaa_asr
 * aaa_error
 * aaa_req_timeout
 * aaa_conn_error
 * aaa_rate_limit
 * aaa_ocs_hold_end
```
in end of `The label type is the Termination Causes types. For Termination causes the following types exist:` section